src/librustdoc/formats/mod.rs

```diff
@@ -0,0 +1,107 @@
+use std::cell::RefCell;
+use std::rc::Rc;
+
+use rustc_span::edition::Edition;
+
+use crate::clean;
+use crate::config::{RenderInfo, RenderOptions};
+use crate::error::Error;
+
+pub trait FormatRenderer: Clone {
```

Comment:
Could you document public items (here, the trait itself and the init function)? That said, I'm fresh in this code base so maybe this convention doesn't apply.